packages/js-sdk/src/Framework.js

```diff
@@ -39,7 +37,7 @@ module.exports = class Framework {
         this.chainId = chainId;
         this.version = version || "test";
         this.resolverAddress = resolverAddress;
-        this.web3 = isTruffle ? global.web3 : new Web3(web3Provider);
+        this.web3 = isTruffle ? global.web3 : web3Provider;
```

Comment:
Note this change

Reply:
kinda breaking change. I will already mark this version as 0.3.0